viewer/src/components/Commands.vue

```diff
@@ -368,7 +374,7 @@ export default class Commands extends Vue {
                 booster,
                 needConfirm: true,
                 tooltip: tiles.boosters[booster].map((spec) => eventDesc(new Event(spec))).join("\n"),
-                warning,
+                warning: warning,
```

Comment:
is this your editor too? (repeated twice below as well)

Reply:
yes, I added that

Reply:
I mean I did that manually..